ProcessLib/GroundwaterFlowFEM.h

```diff
@@ -26,6 +25,8 @@ namespace ProcessLib
 namespace GroundwaterFlow
 {
 
+// TODO now this interface is basically the same for all processes that assemble a
+//      FirstOrderImplicitQuasiLinear ODE system.
```

Comment:
Comment to the comment: I guess the interface is only partially the same because other parameters/values are passed to different processes.